okhttp/src/commonMain/kotlin/okhttp3/internal/http/HttpStatusCodes.kt

```diff
@@ -0,0 +1,203 @@
+/*
+ * Copyright (c) 2022 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package okhttp3.internal.http
+
+/**
+ * The response codes for HTTP, as of version 1.1.
+ */
+
+/**
+ * HTTP Status-Code 200: OK.
+ */
+const val HTTP_OK = 200
```

Comment:
Just noticed this is under a GPL license.  So will delete and import from the same values I guess.

```

 * This code is free software; you can redistribute it and/or modify it
 * under the terms of the GNU General Public License version 2 only, as
 * published by the Free Software Foundation.  Oracle designates this
 * particular file as subject to the "Classpath" exception as provided
 * by Oracle in the LICENSE file that accompanied this code.
 ```

Reply:
fixed